about.html

```diff
@@ -0,0 +1,119 @@
+                interested and passionate in becoming polyglots. <br> Polyglot Language School was created in 2022 by Mr
+                Jonathan Amani Munamire who is himself a Congolese who found it difficult to adapt in another country
+                because of language barriers. <br> That's why Polyglot Language School exists! We are here to help all
+                passionate people to go beyond their limits and to tear down all the languages barriers they may have.
+                <br>
+                With all flexible programs, you can choose a program that meets your expectations! <br> We have more
+                than 45
+                certified teachers and tutors; with a lot of experience.
+                We hope you'll consider joining us this year to build a strong community of learners.
+            </p>
+            <p class="contact">Please Contact us via email for more information <br><b>polyglot.info@gmail.com</b></p>
+            <p class="address">@243 East Section, Hotel2000, Kigali (Rwanda)</p>
+        </section>
+        <section class="second-section" id="logo">
+            <h2>Polyglot Languages School Logo</h2>
+            <hr class="line2">
+            <p>The Logo of Polyglot Languages School was decided through the logo competition from 8.June to 7.July.
+                After online public voting and Selection Committee's judjing process, the logo from a 17years old
+                Rwandan girl won the prize. After consultation with Mr. Jonathan Amani Munamire we optimized the
+                prize-winning-logo for Polyglot Language School as following:</p>
+            <img src="IMG/logo.jpg">
+        </section>
+        <section class="about-section3" id="vision">
+            <h2 class="about-section3-last-head">
+                See our vision and goals
+            </h2>
+
+            <p class="about-section3-paragraph">
+                Take a look at how we work with our students!
+            </p>
+
+            <div class="about-section3-card-div">
+                <div class="about-section3-card">
+                    <p class="about-section3-card-texts">
+                        Different Community Students
+                    </p>
+                </div>
+
+                <div class="about-section3-card2">
+                    <p class="about-section3-card-texts">
+                        Great Studying Programs
+                    </p>
+                </div>
+            </div>
+        </section>
+        <footer>
+            <section class="first-footer">
+                <h2 class="footer-heading">Partners</h2>
+                <hr class="line3">
+                <div class="partners">
+                    <p>KAHOOT</p>
+                    <p>Udemy</p>
+                    <p>COURSERA</p>
+                    <p>Stride</p>
+                    <p>Unacademy</p>
+                    <div>
+            </section>
```

Comment:
- Please **About page** does not contain the partners' section, Kindly remove it. This applies to About page Mobile size as well
- See the original design
- 
![Capture](https://user-images.githubusercontent.com/63972670/196752621-8295832e-b535-456f-9eb2-3836631e51c8.JPG)